src/main/java/de/fraunhofer/aisec/cpg/graph/ValueAccess.java

```diff
@@ -0,0 +1,12 @@
+package de.fraunhofer.aisec.cpg.graph;
+
+/** Expressions can have multiple types of accesses. Determines dataflow (DFG) edges */
+public interface ValueAccess {
+  enum accessValues {
```

Comment:
Is there a specific reason for a non-public enum here? Not sure if I have ever seen this. Makes it a bit annoying to always write `ValueAccess.accesValues.READ`